### PR TITLE
Content edits

### PR DESCRIPTION
**Pull request by jasperan**

### CODE REVIEW

**Comment by jasperan** [comment]
@ttscoff fixed it I believe. We shouldn't have issues with this PR

File: training_and_inference_yolov5/training_and_inference.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+# for yolov5x
+python train.py --img 640 --data ./datasets/y5_mask_model_v1/data.yaml --weights yolov5x.pt --name y5_mask_detection  --save-period 25 --device 0,1 --batch 16 --epochs 3000
+```
+
+...and the model will start training. Depending on the size of the dataset, each epoch will take more or less time. In my case, with 10.000 images, each epoch took about 2 minutes to train and 20 seconds to validate.
+
+> **Note**: training from scratch (no checkpoints) **hugely** decreases training
+{:.notice
+
+}
```
Comment (ttscoff) [inline training_and_inference_yolov5/training_and_inference.md:168]: Can you move this bracket up to the end of line 166 for me, please?

**Review by ttscoff — COMMENTED** [review]
Just one minor change, please

File: training_and_inference_yolov5/training_and_inference.md (hunk 1 above)
Comment (jasperan) [reply training_and_inference_yolov5/training_and_inference.md]: Done

**Review by ttscoff — APPROVED** [review]

**Pull request merged**